cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -1,5 +1,12 @@
 import $ from 'jquery';
 import 'rangeslider.js';
+import {
```

Comment:
Same comment as above; would you consider removing extraneous whitespace changes to reduce the scope of this PR?

Reply:
Whoops! https://github.com/cfpb/cfgov-refresh/pull/3991 appears to have snuck in here. I'll see if I can get that one reviewed and rebase here.

Reply:
This is the non-whitespace diff URL https://github.com/cfpb/cfgov-refresh/pull/3998/files?w=1